src/ducks/apps/components/ApplicationPage/Gallery.jsx

```diff
@@ -3,12 +3,34 @@ import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 
 export const Gallery = ({t, slug, images}) => {
+  if (!images) return null
+  const imagesParts = images.reduce((acc, image, index) => {
+    if (index < 2) {
+      if (!acc.big) acc.big = []
+      acc.big.push(image)
+    } else if (index < 5) {
+      if (!acc.small) acc.small = []
+      acc.small.push(image)
+    }
+    return acc
+  }, {})
   return (
     <div className='sto-app-images'>
-      {images && !!images.length &&
-        images.map((screen, index) =>
-          <img className='sto-app-image' src={screen} alt={`${slug}-image-${index + 1}`} />
-        )
+      {imagesParts.big && !!imagesParts.big.length &&
+        <div className='sto-app-big-images'>
```

Comment:
Since this is a list of images, you should design it as a list (of images) with <ul> / <li> tag